assets/css/edd-admin.css

```
@@ -3516,7 +3516,7 @@ span.edd-to-and-from--separator {
 	#system-info-textarea,
 	#edd-add-discount input[type="text"],
 	#edd-edit-discount input[type="text"],
-	#edd-filters input:not([type='radio']),
+	#edd-filters > input:not([type="radio"]),
```

I'm not sure that change is applying to any inputs anymore. I wonder if this would be better:

```suggestion
	#edd-filters input:not([type="radio"]):not([type="checkbox"]),
```

However, this also kind of has me wondering... what's that line for in the first place? Do we have any input types other than text that we're _actually_ trying to capture here? `number` maybe? Instead of targeting all inputs except radio/checkbox, maybe it's better to only target the ones we actually want.

---

I do think that can safely be removed, as well as the `#edd-filters a` line possibly as well (doesn't do anything since the mobile app link is inside of a `span`).

The `#edd-filters` are also used on the Downloads > Orders screen, and there are some mobile issues there as well (specifically with `#edd-filters #edd-advanced-filters.open .inside`; I think the relative position on `#edd-filters #edd-advanced-filters` is causing problems there), but only one `number` input which might be affected by removing this.

---

I'd like to see the rules removed if possible. There is a lot of CSS fighting itself around these filters, so the more we can strip back the better.